couchdb/source/pkg/apis/sources/v1alpha1/couchdbsource_types.go

```diff
@@ -48,9 +48,21 @@ var _ kmeta.OwnerRefable = (*CouchDbSource)(nil)
 // Check that CouchDbSource implements the Conditions duck type.
 var _ = duck.VerifyType(&CouchDbSource{}, &duckv1beta1.Conditions{})
 
+// FeedType is the type of Feed
+type FeedType string
+
 const (
-	// CouchDbSourceChangesEventType is the CouchDbSource CloudEvent type for changes.
-	CouchDbSourceChangesEventType = "dev.knative.couchdb.changes"
+	// CouchDbSourceUpdateEventType is the CouchDbSource CloudEvent type for update.
+	CouchDbSourceUpdateEventType = "org.apache.couchdb.document.update"
```

Comment:
still, these are coming from a knative component.

how about `dev.knative.couchdb.document.update` ?   we prefix the Kafka/Camel src w/ `dev.knative` as well.  Perhaps scope for a larger discussion ?

Reply:
see the CloudEvents spec (not yet approved): https://github.com/lionelvillard/spec/blob/couchdb/adapters/couchdb.md

The CloudEvents spec should not mention knative.

Reply:
I think the specification itself is not mentioning knative at all